test/quantize_cpu.cpp

```diff
@@ -331,3 +332,100 @@ TEST(quantize_cpu, quantize_to_int8)
     EXPECT_EQ((vector<float>{-127}), read_vector<float>(result_min));
     EXPECT_EQ((vector<float>{127}), read_vector<float>(result_max));
 }
+
+TEST(quantize_cpu, quantizedConv2D_with_relu)
```

Comment:
trying to understand, what's the difference with `quantizedConv2D_fused_relu` apart from Shape and values?

Reply:
None. I was just out of names :)